code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -51,8 +51,7 @@
 	for(var/path in to_spawn)
 		var/amount = to_spawn[path] || 1
 		for(var/i in 1 to amount)
-			var/atom/A = new path(src)
-			A.initialize()  // recursive
```

Comment:
Why did you remove this? iirc this is needed for headsets inside lockers to be properly initialized

Reply:
I did not know that this was needed for headsets, let me bring it back